contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -7681,6 +7944,266 @@ rewrite_column_name_with_omitted_schema_name(T ctx, GetCtxFunc<T> getSchema, Get
 	return "";
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// Spatial Query Helper
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Nit: Should have corresponding closing block at the end.

Reply:
Done